com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -667,23 +672,22 @@ struct InitInfo
 
         internal bool clearAssetsOnVolumeClear = false;
 
-        /// <summary>Delegate for baking state change.</summary>
-        /// <param name="newState">The new baking state.</param>
-        public delegate void BakingStateChangedDelegate(string newState);
-        /// <summary>Delegate called when the baking state is changed. </summary>
-        public BakingStateChangedDelegate onBakingStateChanged;
-
-        /// <summary>The currently selected baking state.</summary>
+        /// <summary>The current baking state.</summary>
         public string bakingState
         {
             get => sceneData.bakingState;
-            set
-            {
-                sceneData.bakingState = value;
+            set => SetBakingState(value, 0.0f);
+        }
+        /// <summary>The current baking state. 1 means state is fully loaded</summary>
+        public float bakingStateLerp => m_BakingStateLerpFactor;
+
+        /// <summary>Set the baking state.</summary>
+        public void SetBakingState(string state, float transitionTime)
```

Comment:
no dircet contrlo on belnded states, only a target and transition time

Reply:
Could we also have a direct blend control? This automatic transition is impossible to control from animation/timeline

Reply:
yep, i'll do it in a second PR though as we would like to merge this one before srp2core and i'll pobably break stuff if i do it now